firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -77,8 +77,13 @@ public class ConfigRealtimeHttpClient {
   @GuardedBy("this")
   private boolean isRealtimeDisabled;
 
-  private final int ORIGINAL_RETRIES = 7;
+  /**
+   * Controls whether or not this client should retry. Does not need to be synchronized because
+   * ordering of thread actions is not important since all connections will timeout after 5 minutes.
+   */
+  private boolean canRetry;
```

Comment:
Discussed offline: rename this to `appIsInBackground` or something similar to better indicate the meaning

Reply:
Updated